corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -143,6 +145,12 @@ def _check_for_exceeded_rate_limits(ip, num, user):
         status, window = _check_for_exceeded_rate_limits(ip_address, number, username)
         if status == _status_accepted:
             _report_usage(ip_address, number, username)
+        else:
+            # log any attempts that are rate limited
+            clean_number = clean_phone_number(number)
```

Comment:
Any reason we clean the phone number here but not elsewhere?

Reply:
Good question. I followed other examples of saving phone numbers so I figured it might be good to clean before logging. Here is an existing example: https://github.com/dimagi/commcare-hq/blob/0c07db5823b4f1b81131a12b4543c3d0c9da8957/corehq/apps/sms/api.py#L523

Reply:
Right forgot to send my reply on this one.

I think since we're rate limiting based on `number`, we should just record `number` and not `clean_phone_number(number)`. If they were ever different (i.e. `number` wasn't already in its normalized form) then calling by calling `clean_phone_number` on it we are only _losing_ information. So yeah I'd keep it simple and store number, which is already a string value.

Reply:
Sounds good. Just pushed.